bin/s2nc.c

```diff
@@ -355,6 +362,10 @@ int main(int argc, char *const *argv)
             s2n_config_disable_x509_verification(config);
         }
 
+        if (!no_session_ticket) {
```

Comment:
nit: double negation is a bit harder to understand, how about renaming to use_session_tickets?

Reply:
Okay

Reply:
you could keep the command line argument the same, but negate the name (and value) of the variable.

Reply:
Okay.